docs/developer/extending/apps/architecture/server-and-client-side-calls.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: Server and client side calls
+---
+
+Usually, apps are built on top of two communication methods - webhooks and API requests.
+While webhooks are always server-side (Saleor core calls the backend of your app),
+API requests from App to Saleor can be both server and client (frontend) sides.
+
+This article highlights main differences of both approaches.
+
+## Two types of tokens
+
+Saleor App API uses two types of tokens:
+
+- AppToken - used for server-side calls
+- AccessToken - used for client-side calls
+
+### AppToken
+
+AppToken is a one-time token generated when the App is being installed. Saleor will call `tokenTargetUrl` from the [App Manifest](../manifest.mdx) and App needs to store the token in the safe place,
+if it wants to consume the Saleor API. Official Saleor Apps and [app-sdk](../developing-apps/app-sdk.mdx) uses [APL concept](./apl.mdx) to store the token.
+
+The AppToken is recommended to use for server-side calls.
+It represents App itself and doesn't store any user information. It doesn't expire, so it should not be exposed to the front end.
+
+### AccessToken
+
+AccessToken is a [JWT token](https://jwt.io/) that represents a user. It is generated once the user opens the App frontend in the Dashboard.
+The Dashboard provides the token to the App using AppBridge.
+
+This token expires, and the dashboard refreshes it in the background. In case of any leak, it will eventually expire and become useless. Hence it's better for client-side calls.
```

Comment:
**polish**: a leak/any leaks

```suggestion
This token expires, and the dashboard refreshes it in the background. In case of a leak, it will eventually expire and become useless. Hence it's better for client-side calls.
```